api/v1alpha/model_aws_region_config_all_of.go

```diff
@@ -0,0 +1,223 @@
+/*
+MongoDB Atlas Administration API
+
+The MongoDB Atlas Administration API allows developers to manage all components in MongoDB Atlas. To learn more, review the [Administration API overview](https://www.mongodb.com/docs/atlas/api/atlas-admin-api/). This OpenAPI specification covers all of the collections with the exception of Alerts, Alert Configurations, and Events. Refer to the [legacy documentation](https://www.mongodb.com/docs/atlas/reference/api-resources/) for the specifications of these resources.
+
+API version: 2.0
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package v1alpha
+
+import (
+	"encoding/json"
+)
+
+// AWSRegionConfigAllOf struct for AWSRegionConfigAllOf
+type AWSRegionConfigAllOf struct {
```

Comment:
This file should not be present as it doesn't have representation in model.
We need to add instruction on how to regenerate model locally (Update readme to run make command that will remove SDK.

Reply:
Fixed